api/src/routing.js

```diff
@@ -427,7 +426,7 @@ app.postJson('/api/v1/places', function(req, res) {
 
 app.postJson('/api/v1/places/:id', function(req, res) {
   auth
-    .check(req, 'can_update_places')
+    .check(req, ['can_edit', 'can_update_places'])
```

Comment:
Change to `hasAllPermissions` (as above).

Reply:
`check` was refactored